tests/00_00_04-change-federation.js

```diff
@@ -0,0 +1,246 @@
+        [KEY_TYPE_MST]: newFederator1PublicKey
+    });
+
+    newFederationKeys.push({
+        [KEY_TYPE_BTC]: newFederator2PublicKey,
+        [KEY_TYPE_RSK]: newFederator2PublicKey,
+        [KEY_TYPE_MST]: newFederator2PublicKey
+    });
+
+    // Sort by btc public key
+    newFederationKeys.sort((keyA, keyB) => keyA.btc.localeCompare(keyB.btc))
+
+    return newFederationKeys;
+
+};
+
+const fundFedChangeAuthorizer = async (rskTxHelper, fedChangeAuthorizerAddress) => {
+    await rskUtils.sendFromCow(rskTxHelper, fedChangeAuthorizerAddress, btcToWeis(0.1));
+};
+
+const getBtcPublicKeysInArray = (btcPublicKeysInString) => {
+    return removePrefix0x(btcPublicKeysInString).match(/.{1,66}/g).map(key => ensure0x(key)) || [];
+};
+
+describe('Change federation', async function() {
+
+    const regtestFedChangeAuthorizer1PrivateKey = REGTEST_FEDERATION_CHANGE_PRIVATE_KEYS[0];
+    const regtestFedChangeAuthorizer2PrivateKey = REGTEST_FEDERATION_CHANGE_PRIVATE_KEYS[1];
+    const regtestFedChangeAuthorizer3PrivateKey = REGTEST_FEDERATION_CHANGE_PRIVATE_KEYS[2];
+
+    let rskTxHelpers;
+    let rskTxHelper;
+
+    let fedChangeAuthorizer1Address;
+    let fedChangeAuthorizer2Address;
+    let fedChangeAuthorizer3Address;
+
+    let initialFederationPublicKeys;
+    let newFederationPublicKeys;
+
+    let initialFederationAddress;
+
+    let bridge;
+
+    let commitFederationEvent;
+  
+    before(async () => {
+
+        rskTxHelpers = getRskTransactionHelpers();
+        rskTxHelper = rskTxHelpers[0];
+
+        // Import the private keys of the federation change authorizers.
+        fedChangeAuthorizer1Address = await rskTxHelper.importAccount(regtestFedChangeAuthorizer1PrivateKey);
+        fedChangeAuthorizer2Address = await rskTxHelper.importAccount(regtestFedChangeAuthorizer2PrivateKey);
+        fedChangeAuthorizer3Address = await rskTxHelper.importAccount(regtestFedChangeAuthorizer3PrivateKey);
+
+        // Send some funds to the federation change authorizers to pay for transaction fees while voting.
+        await fundFedChangeAuthorizer(rskTxHelper, fedChangeAuthorizer1Address);
+        await fundFedChangeAuthorizer(rskTxHelper, fedChangeAuthorizer2Address);
+        await fundFedChangeAuthorizer(rskTxHelper, fedChangeAuthorizer3Address);
+
+        bridge = getBridge(rskTxHelper.getClient(), await getLatestActiveForkName());
+
+        initialFederationPublicKeys = await getCurrentFederationKeys(bridge);
+        newFederationPublicKeys = createNewFederationKeys(initialFederationPublicKeys);
+
+        initialFederationAddress = await bridge.methods.getFederationAddress().call();
+        
+    });
+
+    it('should create a pending federation', async () => {
+
+        // Ensuring no pending federation exists yet.
+        const pendingFederationSize = Number(await bridge.methods.getPendingFederationSize().call());
+        expect(pendingFederationSize).to.be.equal(-1, 'No pending federation should exist yet.');
```

Comment:
Let's move this literal number to a variable or constant that, by its name, represents the result when there is no pending fed.